web/json_object.go

```diff
@@ -144,7 +144,13 @@ func convertJSONObject(entity *framework.EntityConfig, object map[string]any, op
 		complexAttributes = make(map[string]framework.Object, len(complexAttributeFakeEntities))
 
 		for localExternalId, fakeEntity := range complexAttributeFakeEntities {
-			complexValue, ok := object[localExternalId].(map[string]any)
+			// Do not parse null attribute values.
+			value, found := object[localExternalId]
+			if found && value == nil {
+				continue
+			}
```

Comment:
You don't need to recheck that the `localExternalId` key is valid. This is already done above.
https://github.com/SGNL-ai/adapter-framework/pull/4/files#diff-ffee393068a1aa52cd86abae4a7a6c7e0fe4f639153cd3f4e7dd3d7a9d2ee7d1R126-R129
```suggestion
			// Do not parse null attribute values.
			rawComplexValue := object[localExternalId]
			if rawComplexValue == nil {
				continue
			}
```

Reply:
Actually, I would rather do this new check in the first loop above:
https://github.com/SGNL-ai/adapter-framework/pull/4/files#diff-ffee393068a1aa52cd86abae4a7a6c7e0fe4f639153cd3f4e7dd3d7a9d2ee7d1R126-R129
It would be more efficient.